docs/fast_data/tutorials/low-code.mdx

```diff
@@ -26,348 +26,347 @@ To do that, in the console, you will:
 We can create two different endpoints, one for the registry and one for the order.
 
-:::info
-Remember that the endpoint connected to the Fast Data will create just the read endpoint: so, we can't create, update, and delete the data stored in the Projection using a REST request.
+:::caution
+Remember that the endpoint connected to the Fast Data will create just `GET` endpoints: so, we can't perform _update_ and _delete_ operations over the endpoint.
 :::
 
-### Testing Projection
+### Test Projections
 
 Now it's time to [save](/console/handbooks/project-usage.md#save-changes) and [deploy](/console/handbooks/project-usage.md#deploy-changes) the new configuration.
 ![Deploy the configuration](img/10-deploy.png)
 
-After that, we can try to send a message to the Kafka Topic created in the previous steps, and we can see the data stored in the Projection using the APIs available in the [API Documentations](/console/project-configuration/documentation-portal.md).
+After that, we can try to send a message to the _Kafka Ingestion Topics_ created in the previous steps, and we can see the data stored in the Projection using the APIs available in the [API Documentations](/console/project-configuration/documentation-portal.md).
 
-Following are example responses from the API Documentations for Registry and Order Projections after the Kafka messages have been successfully processed.
+Since we have chosen `Debezium` as the format for our _SoR_, the ingestion message that we will send has to comply to [its specifications](/fast_data/inputs_and_outputs.md#debezium).
 
-Here is the response for the Registry Projection after we created a customer with id `reg-1`.
-``` json
+<details>
+<summary>Ingestion Example</summary>
+<p>
+
+We will add one registry and one order to our System of Record, respectively `reg-1` and `ord-1`.
+
+This will mean sending the following ingestion messages to the ingestion topics, using a Kafka Producer:
+
+```json title=demo.development.delivery.registry.ingestion
+{
+  "key": {
+    "registry_id": "reg-1"
+  },
+  "value": {
+    "op": "c",
+    "before": null,
+    "after": {
+      "registry_id": "reg-1",
+      "address": "customer address",
+      "city": "customer city",
+      "email": "customer1@mail.com",
+      "name": "customer 1",
+      "phone": "customer phone",
+      "state": "customer state",
+    }
+  }
+}
+```
+
+```json title=demo.development.delivery.orders.ingestion
+{
+  "key": {
+    "order_id": "ord-1"
+  },
+  "value": {
+    "op": "c",
+    "before": null,
+    "after": {
+      "date": "2024-02-16T11:13:48.322Z",
+      "name": "order-1",
+      "order_id": "ord-1",
+      "price": 123,
+      "registry_id": "reg-1"
+    }
+  }
+}
+```
+
+:::tip
+If you don't have a direct access to your Kafka broker to produce messages, you can use a dedicated microservice such as [Rest2Kafka](/runtime_suite/rest2kafka-service/01_overview.md) to produce Kafka messages as REST request directly from MIA Console. 
+:::
```

Comment:
Probably could be useful to go deeper with the use of the service Rest2Kafka for demo and tutorial cases. 
Maybe we can improve the plug-in documentation to add some example on how to use it.

Reply:
i think it would be out of scope of this tutorial, since in the requirements is assumed to have knowledge and access to kafka. i can remove the tip